snippets/wmco-key-ascii-encoding.adoc

```diff
@@ -0,0 +1,18 @@
+// Text snippet included in the following modules:
+//
+// * modules/configuring-secret-for-wmco.adoc
+// * modules/creating-the-vsphere-windows-vm-golden-image.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+If you created the key pair on a {op-system-base-full} system, before you can use the public key on a Windows system, make sure the public key is saved using ASCII encoding. For example, the following PowerShell command copies a public key, encoding it for the ASCII character set:
```

Comment:
```suggestion
If you created the key pair on a {op-system-base-full} system, before you can use the public key on a Windows system, make sure the public key is saved by using ASCII encoding. For example, the following PowerShell command copies a public key, encoding it for the ASCII character set:
```
IBM Style Word Usage advises to use either "by using" or "that using" to avoid ambiguity: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#using